backport-changelog/readme.md

```diff
@@ -0,0 +1,3 @@
+# Core Backport Changelog
+
+Any PR that makes changes to be backported to core should log a core PR here. It's possible to have multiple Gutenberg PRs link to a single core backport PR. The core backport PR can remain open as long as wanted/needed. The entries are sorted by core release, and each entry should be an md file with the Gutenberg PR number as the file name, and the link to the core backport PR as file content. Files are used to avoid rebase conflicts.
```

Comment:
```suggestion
Any PR that makes changes to be backported to core should log a core PR here. It's possible to have multiple Gutenberg PRs link to a single core backport PR. The core backport PR can remain open as long as wanted/needed. The entries are sorted by core release, and each entry should be an md file with the Gutenberg PR number as the file name, and the link to the core backport PR as file content (see `<core-pr-number>.md` as an example). Files are used to avoid rebase conflicts.
```